src/main/java/salesforce/ui/pages/PageTransporter.java

```diff
@@ -38,6 +38,8 @@ public PageTransporter() {
         urlLightningMap.put("Campaigns Page", PropertiesReader.getInstance().getSalesforceUrl()
                 .getCampaignsLightningUrl());
         urlLightningMap.put("Leads Page", PropertiesReader.getInstance().getSalesforceUrl().getLeadsLightningUrl());
+        urlLightningMap.put("Contracts Page", PropertiesReader.getInstance().getSalesforceUrl()
```

Comment:
You can use constants for this string **"Contracts Page"**

Reply:
All te page is this.